lalrpop-util/src/lib.rs

```diff
@@ -232,6 +232,20 @@ pub struct ErrorRecovery<L, T, E> {
     pub dropped_tokens: Vec<(L, T, L)>,
 }
 
+#[cfg(not(target_os = "windows"))]
+#[doc(hidden)]
+#[macro_export]
```

Comment:
I don't have a ton of knowledge on macro scoping rules, but does this really need `macro_export`?  I thought if it was only used farther down in the same file it wouldn't?  That would allow removing of `doc(hidden)` as well (although I assume I'm just wrong and there's some reason it needs `macro_export`).

Reply:
Yeah, the tricky part here is that this macro doesn't get expanded in the definition of lalrpop_mod in this file, it gets expanded where lalrpop_mod is used which is in the users file. This is why it is exported.

Alternatively, we could create 2 versions of lalrpop_mod, one for the windows case and one other-wise. This would let us hard-code the path separator instead of choosing it via a macro.